proc_IR_noecho.py

```diff
@@ -43,8 +43,8 @@
     ph0 = zeroth_order_ph(s['t2':0],fl=None)
     ph0 /= abs(ph0)
     s /= ph0
-    print(ndshape(s))
-    quit()
+    #print(ndshape(s))
+    #quit()
```

Comment:
adding commented code in! my eyeball hurts!

Reply:
outdated